apps/src/templates/progress/ProgressLesson.jsx

```diff
@@ -39,6 +39,7 @@ const styles = {
   heading: {
     fontSize: 18,
     fontFamily: '"Gotham 5r", sans-serif',
+    cursor: 'pointer'
```

Comment:
Should be viewable on any page that has a `ProgressGroup` (the group is the big purple box, the lesson is the one that says "Lesson 1: something something something". You can also see this in storybook.